static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js

```diff
@@ -4,128 +4,81 @@ define([
-
-        this.canRun = function () {
-            return document.querySelector('#membership-ab-thrasher') &&
-                    !cookies.getCookie('GU_DBPT1M') &&
-                    !userFeatures.isPayingMember() &&
-                    config.page.isFront &&
-                    config.page.pageId.toLowerCase() === "uk" &&
-                    config.page.edition.toUpperCase() === 'UK';
-        };
-
-        this.thrasherContainer = function() {
-            return document.querySelector('#membership-ab-thrasher');
-        };
-
-        this.thrasher = function() {
-            return document.querySelector('.membership-ab-thrasher--wrapper');
-        };
-
-        this.setTitle = function() {
-            if (this.thrasher()) {
-                var titleEl = document.querySelector('.membership-ab-thrasher_header .main_title');
-                if (titleEl) {
-                    titleEl.innerHTML = '<p>Support' + (detect.isBreakpoint({min: 'mobileMedium', max: 'tablet'}) ? ' ' : '<br>')  + 'the Guardian</p>';
-                }
-            }
-        };
-
-        this.setCopy = function() {
-            if (this.thrasher()) {
-                var subTitleEl = document.querySelector('.membership-ab-thrasher_header .sub_title');
-                if (subTitleEl) {
-                    subTitleEl.innerHTML = '<p>We\'re introducing <strong>new ways</strong> to support' + (detect.isBreakpoint({min: 'desktop'}) ? '<br>' : ' ') + 'the Guardian\'s quality journalism and independent voice. Choose to subscribe or contribute today.</p>';
-                }
-            }
-        };
-
-        this.setLink = function(variant) {
-            if (this.thrasher()) {
-                var linkEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button');
-                if (linkEl && linkEl.getAttribute('href')) {
-                    linkEl.setAttribute('href', config.page.membershipUrl + '/bundles?INTCMP=BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.toUpperCase());
-                }
-            }
-        };
-
-        this.setCaption = function() {
-            if (this.thrasher()) {
-                var captionEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button div p');
-                if (captionEl) {
-                    captionEl.textContent = 'Subscribe now';
-                }
-            }
-        };
+    var buildTemplate = function(variant) {
+        return template(contributionsEpicSingleButton, {
+            linkUrl1: config.page.membershipUrl + '/bundles?INTCMP=' + 'BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.id.toUpperCase(),
+            componentName: variant.componentName,
+            title: 'Since you’re here…',
+            p1: '… we’ve got a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And ' +
```

Comment:
Yes, but since then we have refactored the copy into a module, which ensures that the copy we use is consistent across our different tests (see https://github.com/guardian/frontend/pull/16995). To see an example of how it should be used, look at the liveblog design test.